apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/RLSCodeEditor.tsx

```diff
@@ -119,6 +119,12 @@ const RLSCodeEditor = ({
     onChange()
   }
 
+  // when the value has changed, trigger the onChange callback so that the height of the container can be adjusted.
+  // Happens when the value wordwraps and is updated via a template.
+  useEffect(() => {
+    onChange()
+  }, [value])
+
```

Comment:
do you have a screen recording for this issue on prod? i tried to fill up the RLS code editor using statement till it wraps, then click on a template but i dont see any height issue - the code editor goes back to the height of one line correctly

Reply:
The issue happens when you insert a bigger template. The onChange needs to be called so that the height math calculation gets triggered.

https://github.com/supabase/supabase/assets/568291/35dc2985-f19f-4888-878f-3a772a2f749e